docs/faq.rst

```diff
@@ -0,0 +1,115 @@
+monitor to be able to orchestrate nodes integration completely:
+
+ 1. Health Checks must be successful
+
+    The monitor runs periodic health checks with all the nodes registered
+    in the system. Those *health checks* are Postgres connections from the
+    monitor to the registered Postgres nodes, and use the ``hostname`` and
+    ``port`` as registered.
+
+    The ``pg_autoctl show state`` commands column *Reachable* contains
+    "yes" when the monitor could connect to a specific node, "no" when this
+    connection failed, and "unknown" when no connection has been attempted
+    yet, since the last startup time of the monitor.
+
+    The *Reachable* column from ``pg_autoctl show state`` command output
+    must show a "yes" entry before a new standby node can be orchestrated
+    up to the "secondary" goal state.
+
+ 2. pg_autoctl service must be running
+
+    The pg_auto_failover monitor works by assigning goal states to
+    individual Postgres nodes. The monitor will not assign a new goal state
+    until the current one has been reached.
+
+    To implement a transition from the current state to the goal state
+    assigned by the monitor, the pg_autoctl service must be running on
+    every node.
+
+When your new standby node stays in the "catchingup" state for a long time,
+please check that the node is reachable from the monitor given its
+``hostname`` and ``port`` known on the monitor, and check that the
+``pg_autoctl run`` command is running for this node.
+
+The state of the system is blocked, what should I do?
+-----------------------------------------------------
+
+This question is a general case situation that is similar in nature to the
+previous situation, reached when adding a new standby to a group of Postgres
+nodes. Please check the same two elements: the monitor health cheks are
+successful, and the ``pg_autoctl run`` command is running.
+
+The monitor is a SPOF in pg_auto_failover design, how should we handle that?
+----------------------------------------------------------------------------
+
+In the current pg_auto_failover design, there can be a single monitor node.
+That is a *Single Point of Failure*. When the monitor node is not available,
+there can be no state change in the system, which means that no failover can
+happen.
+
+Losing a monitor node when the system is stable has no availability impact,
+as the Postgres streaming replication setup does not depend on the monitor
+to operate normally. If after losing the monitor node then another Postgres
+node becomes unavailable, then a failover will not happen as expected.
+
+So while the impact of losing a monitor node is limited, we still have a
+SPOF in current pg_auto_failover architecture. In the current version of
+things it is expected that a backup and recovery mechanism (PITR) be
+deployed alongside pg_auto_failover, and this should include the monitor.
+
+We have several ideas how to best address this situation in a future version
```

Comment:
Do we really need to include the potential roadmap on the FAQ? We already mention a way to alleviate the issue. I'd rather not include this part. In fact, it'd be more useful if we can explain (or give very basic pointers) on how to do disaster recovery/PITR are noted above.

Reply:
With this comment and Jelte's point that it's not the answer that users are looking for, I have now removed that part. Good point.